tests/providers/test_color.py

```diff
@@ -9,6 +9,7 @@
 from faker.providers.color import RandomColor
 from faker.providers.color.fa_IR import Provider as FaIrProvider
 from faker.providers.color.hy_AM import Provider as HyAmProvider
+from faker.providers.color.es_ES import Provider as EsEsProvider
```

Comment:
Can yo osrt these imports alphabetically? IE: `faker.providers.color.es_ES` should come above `faker.providers.color.fa_IR`